draft-ietf-quic-transport.md

```diff
@@ -2544,6 +2562,12 @@ coordinate the creation of streams; they are created unilaterally by either
 endpoint.  Endpoints can use acknowledgments to understand the peer's subjective
 view of stream state at any given time.
 
+Only the initiator of a stream may send a STREAM frame without receiving one
+unless otherwise specified by the application.  All implicitly opened streams
```

Comment:
This sentence is a bit confusing. I don't think we want to allow applications to override stream ID semantics, do we?

Reply:
Not sure, I copied this language from Ian's PR on this topic. Perhaps I should just nuke the whole paragraph?

Reply:
Yes, this was an effort to fix a problem with my design that you do not have, so I'd nuke it.

Reply:
Done.